MovieQuizUITests/AccessibilityIdentifier.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  AccessibilityIdentifier.swift
+//  MovieQuizUITests
+//
+
+enum AccessibilityIdentifier {
+    enum MainScreen {
+        static let view = "View"
+        static let yesButton = "YesButton"
+        static let noButton = "NoButton"
+        static let posterImageView = "PosterImageView"
+        static let questionIndexLabel = "QuestionIndexLabel"
+        static let activityIndicator = "ActivityIndicator"
+    }
+    
+    enum Alert {
+        static let view = "AlertView"
+    }
+}
```

Comment:
Руководствовался этим https://stackoverflow.com/a/38585994
`The advantage of using a case-less enumeration is that it can't accidentally be instantiated and works as a pure namespace`